front/app/containers/Admin/projects/project/poll/PollAnswerOptions/OptionForm.tsx

```diff
@@ -65,26 +55,19 @@ const OptionForm = ({ question, collapse }: Props) => {
 
   return (
     <Container key={question.id} id={question.id} className="e2e-options-form">
-      <Row>
-        <DisabledDragHandle>
-          <Icon name="sort" />
-        </DisabledDragHandle>
-        <TextCell className="expand">
-          <b>
-            <FormattedMessage
-              {...messages.optionsFormHeader}
-              values={{
-                questionTitle: (
-                  <Text display="inline" fontWeight="normal">
-                    <T value={question.attributes.title_multiloc} />
-                  </Text>
-                ),
-              }}
-            />
-          </b>
-        </TextCell>
-      </Row>
-      <OptionsContainer>
+      <Title variant="h4" color="primary">
+        <FormattedMessage
+          {...messages.optionsFormHeader}
+          values={{
+            questionTitle: (
+              <Text display="inline" fontWeight="normal">
```

Comment:
Our `Title` will render a `h4` element, while the `Text` will render a `p` element. This is technically invalid HTML: https://stackoverflow.com/a/19779520

You can instead consider using `<Text as="span"`. However since React doesn't consider it bad enough for an invalid DOM nesting error, and since it has no visual effect I wouldn't consider it blocking

Reply:
Sure, I'll change it. However, I doubt polling is used much (if at all).